docs/operations/runbooks/rotate_application_secrets.md

```diff
@@ -20,14 +20,14 @@ Where `credentials-<ENVIRONMENT>.json` looks like:
 
 You can see the current environment with `cf env <APP>`, for example `cf env getgov-unstable`.
 
-The command `cups` stands for [create user provided service](https://docs.cloudfoundry.org/devguide/services/user-provided.html). User provided services are the way currently recommended by Cloud.gov for deploying secrets. The user provided service is bound to the application in `manifest-<ENVIRONMENT>.json`.
+The commands `cups` and `uups` stand for [`create user provided service`](https://docs.cloudfoundry.org/devguide/services/user-provided.html) and `update user provided service`. User provided services are the way currently recommended by Cloud.gov for deploying secrets. The user provided service is bound to the application in `manifest-<ENVIRONMENT>.json`.
 
 To rotate secrets, create a new `credentials-<ENVIRONMENT>.json` file, upload it, then restage the app.
 
 Example:
 
 ```bash
-cf cups getgov-credentials -p credentials-unstable.json
+cf create-user-provided-service getgov-credentials -p credentials-unstable.json
```

Comment:
**question:** Will this fail if the service is already created? Do we need to mention `update-user-provided-service` here just in case?